ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -364,7 +364,9 @@ export function D3Blueprint(container, options) {
     function onKeyUp() {
         d3.event.stopPropagation();
```

Comment:
Isn't the usual pattern to stop propagation only if the event is handled?

Happy to leave like this but one to watch if there are any related issues.

Reply:
No part of this change-set.